web/themes/custom/urbanobserver/templates/block/block--urbanobserver-sociallinks-2.html.twig

```diff
@@ -0,0 +1,19 @@
+{% set heading_id = attributes.id ~ '-menu'|clean_id %}
+<nav role="navigation" aria-labelledby="{{ heading_id }}"{{ attributes|without('role', 'aria-labelledby') }}>
+  {# Label. If not displayed, we still provide it for screen readers. #}
+  {% if not configuration.label_display %}
+    {% set title_attributes = title_attributes.addClass('visually-hidden') %}
+  {% endif %}
+  {% set config = config_pages_field('global_configurations', 'field_social_links_text', 'default') %}
+  {% if config[0] %}
```

Comment:
why?

Reply:
First of all, we get a config from our config page, from the field social links, then to create <div> we need a condition. Here, if config has a child, because [0] means first child, we create a div. We should be sure that config is not empty to display smth, not just empty <div>

Reply:
> First of all, we get a config from our config page, from the field social links, then to create div we need a condition. Here, if config has a child, because [0] means first child, we create a div. We should be sure that config is not empty to display smth, not just empty div

do you need [0]?

Reply:
Yes I do. Because just config returns me value anyway because by default even empty config has cache, context fields etc. That is why I need [0] to return me a certain element.